Makefile

```diff
@@ -116,29 +116,37 @@ format:
 
 test: test_python_coverage test_python_unit_coverage test_python_property test_compilation test_binary_size test_specs test_installation test_apps
 
-test_python:
-	$(PYTEST) -m "not hypothesis" tests
+test_python: test_python_unit test_python_integration test_python_compilation prove_python_tests test_python_tools test_python_ide
 
 test_python_unit:
 	$(PYTEST) tests/unit
 
 test_python_integration:
 	$(PYTEST) tests/integration
 
+test_python_compilation:
+	$(PYTEST) tests/compilation
+
 test_python_property:
-	$(PYTEST) -m "not verification" tests/property
+	$(PYTEST) tests/property
+
+test_python_tools:
+	$(PYTEST) tests/tools
+
+test_python_ide:
+	$(PYTEST) tests/ide
 
 test_python_property_verification:
-	$(PYTEST) -m "verification" -s tests/property
+	$(PYTEST) tests/property tests/property_verification
```

Comment:
`test_python_property_verification` now also executes all non-verification tests, which it did not do before. This also means these tests are executed twice in CI now.

Reply:
Fixed.